tree/dataframe/inc/ROOT/RSqliteDS.hxx

```diff
@@ -113,6 +113,9 @@ protected:
    Record_t GetColumnReadersImpl(std::string_view name, const std::type_info &) final;
 };
 
+RDataFrame FromSqlite(std::string_view fileName, std::string_view query);
```

Comment:
I don't know whether we should take the chance to conform to its official naming: https://www.sqlite.org/index.html

Reply:
absolutely, thank you

Reply:
ah, there is the incredibly annoying problem that the header to include is called `RSqliteDS.hxx` :man_facepalming: